src/cloudant/__init__.py

```diff
@@ -13,10 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-_cloudant_
-
-Cloudant / CouchDB Python Client API
-
+Cloudant / CouchDB Python client library API package
```

Comment:
We added a section to the java docs about [compatibility](http://static.javadoc.io/com.cloudant/cloudant-client/2.0.0/overview-summary.html#Compatibility) perhaps something similar would be useful in these docs rather than saying "Cloudant / CouchDB".